docs/next/components/mdx/includes/dagster/DagsterDevTabs.mdx

```diff
@@ -20,20 +20,20 @@ dagster dev -f my_file.py -f my_second_file.py
 </TabItem>
 <TabItem name="From a module">
 
-Dagster can also load Python modules as [code locations](/concepts/code-locations). When this approach is used, Dagster loads the definitions defined at the top-level of the module, in a variable containing the <PyObject object="Definitions" /> object of its root `__init__.py` file. As this style of development eliminates an entire class of Python import errors, we strongly recommend it for Dagster projects deployed to production.
+Dagster can also load Python modules as [code locations](/concepts/code-locations). When this approach is used, Dagster loads the definitions defined in the module passed to the command line. It is recommended to define a variable containing the <PyObject object="Definitions" /> object in a submodule named `definitions` inside the Python module. In practice, the submodule can be created by adding a file named `definitions.py` at the root level of the Python module. As this style of development eliminates an entire class of Python import errors, we strongly recommend it for Dagster projects deployed to production.
```

Comment:
```suggestion
Dagster can also load Python modules as [code locations](/concepts/code-locations). When this approach is used, Dagster loads the definitions defined in the module passed to the command line.

We recommend defining a variable containing the <PyObject object="Definitions" /> object in a submodule named `definitions` inside the Python module. In practice, the submodule can be created by adding a file named `definitions.py` at the root level of the Python module.

As this style of development eliminates an entire class of Python import errors, we strongly recommend it for Dagster projects deployed to production.
```

Let's break this up a bit - the paragraph is pretty visually dense and there's a lot to take in.

Reply:
Updated in [32844cf](https://github.com/dagster-io/dagster/pull/23316/commits/32844cfc77898e48eb467649e4206d343f619b4f)